understanding/20/contrast-minimum.html

```diff
@@ -13,15 +13,14 @@ <h1>Understanding Contrast (Minimum)</h1>
       <h2>Intent of Contrast (Minimum)</h2>
       
       
-      <p>The intent of this Success Criterion is to provide enough contrast between text and
-         its background so that it can be read by people with moderately low vision (who do
-         not use contrast-enhancing assistive technology). For people without color deficiencies,
-         hue and saturation have minimal or no effect on legibility as assessed by reading
-         performance (Knoblauch et al., 1991). Color deficiencies can affect luminance contrast
-         somewhat. Therefore, in the recommendation, the contrast is calculated in such a way
-         that color is not a key factor so that people who have a color vision deficit will
-         also have adequate contrast between the text and the background.
-      </p>
+	<p>The intent of this Success Criterion is to provide enough contrast between text and its background so that it can be read by people with moderately low vision.
+    This user group does not typically use contrast-enhancing assistive technology. 
+    For people without color deficiencies, hue and saturation have minimal or no effect on legibility as assessed by reading performance (Knoblauch et al., 1991).
+    Color deficiencies can affect luminance contrast somewhat. 
```

Comment:
reads a bit odd/informal with the "somewhat" at the end. Maybe "Color deficiencies can have a slight impact on the perception of luminance contrast"

Reply:
I agree about the "somewhat" at the end.  How about using parallel of previous sentence?  So instead of:

> Color deficiencies can affect luminance contrast somewhat. 

We would have:

> For many people with color vision deficiencies, luminance contrast has a significant impact on legibility.

Reply:
I feel the impact of luminance difference is on all people and not specific to color vision deficiencies. To the contrary, people with color vision impairments often have a very big luminance sensitivity.

How do you like the following:
The intent of this Success Criterion is to provide enough contrast between text and its background so that it can be read by people with moderately low vision.
    This user group does not typically use contrast-enhancing assistive technology. 
For many people, luminance contrast has a significant impact on legibility.
    For people without color deficiencies, hue and saturation have minimal or no effect on legibility as assessed by reading performance (Knoblauch et al., 1991).
    Yet color deficiencies can also affect luminance contrast.

Reply:
Hi @gundulaniemann 

> _I feel the impact of luminance difference is on all people and not specific to color vision deficiencies._

As far as **reading and readability** is concerned, it's all about **luminance** contrast, for all sighted users.

The visual word form area (VWFA) is the pre-lexical processing that recognizes whole words in the letter pears, and it functions on the luminance channel exclusively without consideration of color as a hue or colorfulness. MRI studies using the Stroop test confirm this (as does the Stroop test itself for that matter).

**_All vision types_** require ample luminance contrast for best readability. By "ample" I mean what Ian Bailey/Jan Lovi-Kitchin call the "contrast reserve", for best fluent readability, they suggest 20 times the threshold (JND).


> _To the contrary, people with color vision impairments often have a very big luminance sensitivity._

Yes, the deutan/protan/tritan forms of color vision deficiency (CVD) have as good or better contrast sensitivity as standard vision. The exception is protan and red/orange against black.

I am going to respond to this more in depth in the later post.